### ISSUE TITLE
Run Once For Current File Setting Not Working

### ISSUE THREAD

**Issue by ort-cloud** [wallabyjs/quokka#405]
### Issue description or question
Set file to only run the one time. Keeps updating after each change.

### Code editor version

Visual Studio Code v1.38.0

### OS name and version

Windows

**Comment by smcenlly** [comment]
Thanks for reporting the problem. We duplicated an issue with `Run Once for Current File` refreshing line indicators and live value messages with the value of the previous execution. We believe that this is the issue that you experienced. This meant that your file was only actually running once. You can confirm this behavior using something like `new Date() /*?*/` and you will see that the date does not change.

We have fixed this issue in the latest version of Quokka.js for VS Code, v1.0.241.

**Comment by ort-cloud** [comment]
I understand you closed this issue, but the problem still persists. I have made sure I have the most current version. I have uninstalled and reinstalled. If I select `Run Once for Current File` it still updates every change made. For example if I write`setTimeout(() => {
  console.log('Hey! I waited 5 seconds.');
}, 5000);` then select `Run Once for Current File` I will get the output expected of `Hey! I waited 5 seconds.`. If I change the code to `setTimeout(() => {
  console.log('Hey! I waited 6 seconds.');
}, 5000);` Quokka then updates my output to say `Hey! I waited 6 seconds.`. Correct me if I am wrong on the intended functionality, but shouldn't it only run the operation once then stop running operations until I ask Quokka to run again?
